djangoProject/users/views.py

```diff
@@ -21,6 +21,21 @@ class UserRegistrationView(generics.CreateAPIView):
     serializer_class = UserRegistrationSerializer
     permission_classes = [AllowAny]
 
+    def create(self, request, *args, **kwargs):
+        # Serialize and validate the data
+        serializer = self.get_serializer(data=request.data)
+        serializer.is_valid(raise_exception=True)
+
+        # Save the user
+        user = serializer.save()
+        role = user.user_type
+
+        # Redirect based on the role
+        if role == "startup" or role == "both":
```

Comment:
Instead of combining `startup` and `both` in the same condition, explicitly handle them to avoid future ambiguity.